docs/language.rst

```diff
@@ -1757,6 +1757,60 @@ The syntax for calling external call is the same as the external call, except fo
 that it must be done on a contract type variable. Any error in an external call can
 be handled with :ref:`try-catch`.
 
+Internal calls and externals calls
+___________________________________
+
+An internal function call is one that is executed by the current contract. This
+is much more efficient than an external call. An external call requires the
+address of the contract to call, the arguments must be *abi encoded* (also known
+as serialization), then the runtime must set up the VM for the called contract
+(the callee). It must decode the arguments, and then encode return values. Lastly
+the first contract (the caller) must decode return values.
+
+Whenever a call is done on a contract type, this will intiate an external call.
```

Comment:
```suggestion
A call done on a contract type will always initiate an external call.
```

Reply:
All good, I've amened the commit with your suggestion